src/connector/src/parser/debezium/avro_parser.rs

```diff
@@ -116,6 +126,20 @@ impl DebeziumAvroParserConfig {
         }
     }
 
+    pub fn get_pk_names(&self) -> Result<Vec<String>> {
+        Self::get_pk_names_inner(&self.key_schema)
+    }
+
+    pub(crate) fn get_pk_names_inner(key_schema: &Schema) -> Result<Vec<String>> {
+        if let Schema::Record { fields, .. } = key_schema {
+            Ok(fields.iter().map(|field| field.name.clone()).collect_vec())
+        } else {
+            Err(RwError::from(InternalError(
+                "inner avro key schema invalid, record required".into(),
```

Comment:
the error looks strange to me, may refer to the top level message should be a `record`